### ISSUE TITLE
Is the data gone? 

### ISSUE THREAD

**Issue by beanumber** [beanumber/airlines#59]
Our worst nightmare...

```
Downloading flight data...
Downloading 2 new files.  0 untouched.
trying URL 'http://tsdata.bts.gov/PREZIP/On_Time_On_Time_Performance_1987_11.zip'
Error in download.file(url, method = method, ...) : 
  cannot open URL 'http://tsdata.bts.gov/PREZIP/On_Time_On_Time_Performance_1987_11.zip'
```

**Comment by beanumber** [comment]
Nevermind, I think I have a fix.

**Comment by nicholasjhorton** [comment]
Speaking of which, do you have a cached copy of these data?  If they did happen to disappear, that would allow us to figure out a way to make them available in some other manner.

**Comment by beanumber** [comment]
I have my local store, but that isn't a robust solution.

**Comment by fhoffa** [comment]
What was the solution? http://tsdata.bts.gov/ is not replying to me :(

**Comment by fhoffa** [comment]
4 days later, none of these urls work:

- https://transtats.bts.gov/Prezip/On_Time_On_Time_Performance_1987_11.zip (404)
- http://tsdata.bts.gov/PREZIP/On_Time_On_Time_Performance_1987_11.zip (timeout)

**Comment by beanumber** [comment]
The data itself seems to be unavailable. If you go here:

https://www.transtats.bts.gov/DL_SelectFields.asp?Table_ID=

and click on "Prezipped File", that will generate the URL. But all of the months I tried generated 404 errors. 

I filed a request. I don't think there is anything I can do until they fix the site.

**Comment by beanumber** [comment]
Strangely, the URL from #37 still works! I pushed a fix in e2de1f8b. 

This should be working now, even though their website is not.
